### DISCUSSION TITLE
Sep 27 Office Hours

### DISCUSSION THREAD

**Discussion by michaelfig** [Agoric/agoric-sdk#8395] [Office Hours]
[Meeting recording](https://agoric.zoom.us/rec/share/ZYU0onNa2CjfJnjEtCyTtELwbZq0-MidGsPabxlSauV3-KEp5iGiwO9zC_E0sTmJ.3APSrRsQNaxx4Rl1), Passcode: !H98xWny

Questions about:
- non-durable governance implications (can't pause invitation makers after upgrade)
- finding a place to store contract-specific parameters (like privateArgs) in the core-eval promise spaces
```js
  consume: {
    durableMapStore: durableMapStoreP,
  },
  produce:  {
    durableMapStore: resolveDurableMapStore,
  }
// ...
let durableMapStore;
try {
  durableMapStore = rootZone.mapStore('durableMapStore');
  durableMapStore.resolve(durableMapStore);
} catch (e) {
  durableMapStore = await durableMapStoreP;
}
const zone = makeDurableZone(durableMapStore).subZone(contractInfo.storagePath);
// proceed as before
```

**Comment by anilhelvaci** [comment]
https://github.com/Agoric/agoric-sdk/discussions/8303
